### ISSUE TITLE
Add high-contrast color themes

### ISSUE THREAD

**Issue by ERSUCC** [NetLogo/NetLogo#2707]
Both high-contrast light and high-contrast dark themes should be added, primarily for improved accessibility. This is mostly a design problem, the infrastructure is already there to support new color themes.
